frontend/src/app/shared/restaurants.service.ts

```diff
@@ -31,7 +31,9 @@ export class RestaurantsService {
         lat: restaurant.location.lat,
         lng: restaurant.location.lng,
         photos: photos,
-        photo: photos[0]
+        photo: photos[0],
+        routeDistance: restaurant.route.distance + "m",
+        routeDuration: (restaurant.route.duration / 60) + "min"
       }
```

Comment:
the service is not responsible for formating  the routeDistance.
you can create a angular pipe that we can use later .
angular pipes are like unix pipe . 
example 
 `{{restaurant.route.distance|m}}`
[
https://angular.io/guide/pipes](
https://angular.io/guide/pipes)